verify-push-backend/assets/index.html

```diff
@@ -255,30 +311,41 @@ <h3>Factors</h3>
         identity: $("#identity").val(),
       });
 
-      digestMessage($("#identity").val())
-        .then((identity) => {
-          const data = new URLSearchParams();
-          data.append("identity", identity);
-          data.append("message", $("#message").val());
-          data.append("factor", $("#factor").val());
+      const identity = $("#identity").val()
+      const data = new URLSearchParams();
+      data.append("identity", identity);
+      data.append("message", $("#message").val());
+      data.append("factor", $("#factor").val());
 
-          var details = $(".detail")
-            .toArray()
-            .map((detail) => detail.value)
-            .reduce(function (result, value, index, array) {
-              if (index % 2 === 0) result.push(array.slice(index, index + 2));
-              return result;
-            }, []);
+      var hiddenDetails = $(".hidden-detail")
+        .toArray()
+        .map((hiddenDetail) => hiddenDetail.value)
+        .reduce(function (result, value, index, array) {
```

Comment:
since this is used in two places now (for `details` and `hiddenDetails`) we could pull this out into a separate function.

Reply:
Done